### DISCUSSION TITLE
How bad could it be just to ignore hmac validation?

### DISCUSSION THREAD

**Discussion by luis-guideti** [Auties00/WhatsappWeb4j#240] [General]
(discussion)
I keep having this issue with hmac which is very annoying, some days I can login on first try, some on third, and some I just cant, I can delete all conversations and everything and its impossible to login. After logged in everything works fine but this inconsistency is very annoying and is always a chore to login.

So I just went ahead and removed the two hmac validations that where giving me trouble since the returns values are never used anyways... but I am wondering, how bad could this be?

**Comment by Auties00** [comment]
It's probably a very bad idea, can you show me your code to determine what you are doing? Also what version are you using?

**Comment by luis-guideti** [comment]
I´ve been running it ignoring all errors since I opened this conversation, lots and lots of messages and conversations, a few eventualy pop up with this error but at least I dont have to reconnect again, this library is just great I love it.
